packages/alea-frontend/pages/u/[institution].tsx

```diff
@@ -246,3 +245,28 @@ const StudentHomePage: NextPage = () => {
 };
 
 export default StudentHomePage;
+export async function getStaticPaths() {
+  const paths = Object.keys(UniversityDetail).map((key) => {
+    return { params: { institution: key } };
+  });
+  return {
+    paths,
+    fallback: true,
```

Comment:
There are no other universities. `fallback` should be false, so that we return 404 for unknown university